packages/connect/package.json

```diff
@@ -15,25 +18,62 @@
     "url": "https://github.com/paritytech/substrate-connect/issues"
   },
   "homepage": "https://github.com/paritytech/substrate-connect#readme",
+  "scripts": {
+    "prebuild": "yarn clean",
+    "test-trace": "node --experimental-vm-modules --trace-warnings node_modules/.bin/jest dist",
+    "test": "node --experimental-vm-modules node_modules/.bin/jest dist",
```

Comment:
We need a `pretest` script to run the build or this will fail on a fresh clone (and run the tests against out-of-date built files potentially)